openmp/libomptarget/plugins-nextgen/amdgpu/dynamic_hsa/hsa_ext_amd.h

```diff
@@ -63,6 +63,7 @@ typedef enum {
 } hsa_amd_memory_pool_access_t;
 
 typedef enum hsa_amd_agent_info_s {
+  HSA_AMD_AGENT_INFO_CHIP_ID = 0xA000,
```

Comment:
Just checking, is this a new thing? I remember having issues when something came from a really new HSA library, I think from the SDMA patch.

Reply:
It's not, I believe. The low value should be indicative.